chore: add callouts and troubleshooting guide for configuration (#6967)
aws-amplify/docs: src/fragments/lib/analytics/native_common/getting-started/30_initAnalytics.mdx

Amplify.configure(amplifyconfig);
```

> <Callout warning="true">

Reviewer: I know fragments are banned, but this seems like a perfect use for one -_-

Author: Yeah, I checked with René initially and he affirmed that we should not be using them even in this case.

> Make sure you call `Amplify.configure` as early as possible in your application. A `Library Not Configured` error is thrown if Amplify.configure isn't called before other Amplify JavaScript APIs. Review the [Library Not Configured Troubleshooting guide](/[platform]/build-a-backend/troubleshooting/library-not-configured/) for possible causes of this issue.

Reviewer: Some nits, feel free to disregard

```suggestion
Make sure you call `Amplify.configure` as early as possible in your application's life-cycle. A `Library Not Configured` error is thrown if `Amplify.configure` has not been called before other Amplify JavaScript APIs. Review the [Library Not Configured Troubleshooting guide](/[platform]/build-a-backend/troubleshooting/library-not-configured/) for possible causes of this issue.
```

Author: Nice, definitely better.
